lib/assert.js

```diff
@@ -60,11 +60,21 @@ function isSWEnv() {
     self instanceof ServiceWorkerGlobalScope);
 }
 
+function isValue(object, expectedValue) {
+  const parameter = Object.keys(object).pop();
```

Comment:
I've had one or two situations where I've avoid the assert library because the object wrapping resulted in some weird issues :(